gnucash/import-export/ofx/gnc-ofx-import.cpp

```diff
@@ -1199,23 +1203,20 @@ gnc_ofx_process_next_file (GtkDialog *dialog, gpointer user_data)
     else
     {
         // Final cleanup.
-        delete info;
+        info.reset();
```

Comment:
Not necessary, info take care of itself when it goes out of scope.

Reply:
> ~Not necessary, info take care of itself when it goes out of scope.~ Edit: Never mind, the problem is bigger than that.

Yups. Hence my preference to carefully commit small changes, and perform the bigger changes as a separate PR.